jekyll/_includes/head.html

```diff
@@ -6,7 +6,10 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
 	<meta name="description" content="{% if page.description %}{{ page.description | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 	{% if page.redirect %}<meta http-equiv="refresh" content="0; url={{ page.redirect }}">{% endif %}
-	<link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
+  {% assign current_url = page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url %}
+	<link rel="canonical" href="{{current_url}}">
+  <link rel="alternate" hreflang="{{page.lang}}" href="{{current_url}}" />
+  {% if page.lang == "en" %}<link rel="alternate" hreflang="ja" href="{{ current_url | replace: '/docs/','/docs/ja/' }}" />{% elsif page.lang == "ja" %}<link rel="alternate" hreflang="en" href="{{ current_url | replace: '/docs/ja/','/docs/' }}" />{% endif %}
```

Comment:
Formatting is not ideal. I added two items to the blue tape list to address this soon: https://circleci.atlassian.net/wiki/spaces/GD/pages/6325928038/Blue+Tape+List

Reply:
Fine for now! I usually drop stuff like this into multiple lines indented by two spaces for each if/else block.

Reply:
I can do that too 👍